Tests/UpstreamIntegrationTests/util/TestSetupHelpers.swift

```diff
@@ -0,0 +1,53 @@
+//
+// Copyright 2023 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License. You may obtain a copy
+// of the License at http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under
+// the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+// OF ANY KIND, either express or implied. See the License for the specific language
+// governing permissions and limitations under the License.
+//
+
+import Foundation
+
+/// Extracts Edge location hint from shell environment. If nothing is set the default value `nil` is returned
+ func getLocationHint() -> EdgeLocationHint? {
+    let edgeLocationHint = EdgeLocationHint()
+    return edgeLocationHint
+}
+
+/// Extracts Edge enviroment from shell environment. If nothing is set the default value `prod` is returned
+func getEdgeEnvironment() -> EdgeEnvironment {
+    let edgeEnvironment = EdgeEnvironment()
+    print("Using Edge Network environment: \(edgeEnvironment.rawValue)")
+
+    return edgeEnvironment
+}
+
+/// Creates a valid interact URL using the provided location hint.
+/// - Parameters:
+///    - locationHint: The location hint String to use in the URL
+/// - Returns: The interact URL with location hint applied
+public func createInteractUrl(with locationHint: String?) -> String {
```

Comment:
Since these are test setup helpers based on the input params, I think this one should go to TestBase+EdgeHelpers

Reply:
I kept this here since it doesn't deal with events (Extension related) and is purely string helper.

Reply:
But I see your point. I will move it.